AirCasting/Dashboard/SessionListView.swift

```diff
@@ -83,7 +83,7 @@ struct SessionsListView: View {
                 }
             }
             .coordinateSpace(name: listCoordinateSpaceName)
-            .background(Color.aliceBlue)
+            .background(Color.aircastingWhite)
```

Comment:
Can this `aircastingWhite` color be replaced with `secondaryBackground`? Aren't they the same?

Reply:
(I mean replacing it and deleting in general, not just here)

Reply:
Unfortunately, these colors are different a little bit.